packages/shared/src/components/Feed.tsx

```diff
@@ -144,6 +142,14 @@ export default function Feed<T>({
       variables,
     );
   const { ranking } = (variables as RankVariables) || {};
+  const {
+    onOpenModal,
+    onCloseModal,
+    onPrevious,
+    onNext,
+    article,
```

Comment:
Looking at it again, we should consider renaming this variable.
Suggestion: `selectedArticle` / `selectedPost`